contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -33,6 +33,7 @@ var ShareView = BaseViews.BaseView.extend({
         this.container = options.container;
         this.current_user = options.current_user;
         this.originalData = this.model.toJSON();
+        this.show_list = this.show_list();
```

Comment:
This seems redundant. I'm not seeing much of a change in functionality, but I might not be testing it right either.